src/app/components/Recommendations/RecommendationsItem/index.test.tsx

```diff
@@ -18,6 +18,26 @@ describe('RecommendationsItem', () => {
     );
   });
 
+  it('should not render an image if critical metadata does not exist', () => {
+    const recommendationWithoutImage = {
+      ...recommendationsFixtures[0],
+      image: {
+        locator: '',
+        originCode: '',
+        altText: '',
+      },
```

Comment:
Is this how the data is being returned from the BFF for Most Read Recommendations? Should we consider doing some cleanup in the BFF?

Reply:
This is how the image object is returned from the BFF. What type of cleanup in the BFF would you recommend? We'd still need to check in Simorgh for the existence of the image object, I guess it could be simplified to `if image` but that would be it?

Reply:
Yeah I had though perhaps image should be returned as a null object, rather than an object with empty values for everything.

Reply:
Yea I'll maybe do that as CPS data is structured differently to Optimo, so performing checks on the likes of `locator` and `originCode` isn't the same across the board.

Reply:
Thanks. Ideally(?) the BFF would just return an image src so that we don't have to cater for these locators + origin codes, but that's part of the wider discussion about where the image logic should truly live.